dn1/scripts/demo.jl

```diff
@@ -0,0 +1,197 @@
+using Graphs
+using dn1
+using Plots
+
+#' ## 18.1.1 SOR iteracija za razpršene matrike
+
+#' Igor Nikolaj Sok, nummat 2025
+
+#' V sklopu te naloge smo morali implementirati podatkovni tip RazpršenaMatrika(V, I), ki hrani razpršeno matriko velikosti nxn s pomočjo matrik V in I velikosti nxm, kjer je m<=n in velja A[i, I[i, j]] = V[i, j]
```

Comment:
Enačbe lahko ovijete v $, kodo pa v `, da so v PDF dokumentu lepše prikazani:
```suggestion
#' V sklopu te naloge smo morali implementirati podatkovni tip `RazpršenaMatrika(V, I)`, ki hrani razpršeno matriko velikosti nxn s pomočjo matrik $V$ in $I$ velikosti $n\times m$, kjer je $m\le n$ in velja `A[i, I[i, j]] = V[i, j]`.
```